components/hacbs/release-service/overlays/kcp-stable/apiexport.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: release-service
```

Comment:
```
$ kustomize build components/hacbs/release-service/overlays/kcp-stable/
Error: no matches for Id apis.kcp.dev_v1alpha1_APIExport|release-service|release-service; failed to find unique target for patch apis.kcp.dev_v1alpha1_APIExport|release-service
```

APIExport is named 'release' in the release-service project

Reply:
another good catch. changing it in the release-service project to be `release-service`

Reply:
changed in https://github.com/redhat-appstudio/release-service/pull/116